lib/graphql/metrics/analyzer.rb

```diff
@@ -32,17 +32,17 @@ def on_enter_operation_definition(_node, _parent, visitor)
       end
 
       def on_leave_field(node, _parent, visitor)
-        return if visitor.field_definition.introspection?
+        return if visitor.field_definition.graphql_definition.introspection?
```

Comment:
It would be good to figure out exactly what the upstream issue is here. Meaning was this a regression and the method should be expose, or is this the proper way to access it now?

Reply:
Good question! cc: @rmosolgo

Reply:
It seems like graphql-ruby should expose `visitor.field_definition.introspection?` ?

Reply:
Yeah, it's a regression. `Schema::Field` needs `def introspection?; @introspection; end`

Reply:
PR incoming

Reply:
https://github.com/rmosolgo/graphql-ruby/pull/2892